### DISCUSSION TITLE
Receiving an error: malloc(): invalid size (unsorted)

### DISCUSSION THREAD

**Discussion by jfpe37** [firemodels/fds#13275] [General]
Good Morning
I received the attached error "**malloc(): invalid size (unsorted)**", After 214.8 seconds.
(SPRK start to work in 76.2 sec, and Blowers start to work in 91.25 sec)
I am attaching the FDS file after I removed the objects of the structure due to the need to maintain confidentiality.
I run on a **linux** with 256 cores and 1 terabyte of memory.
(Attaches the data).
I have similar run on (a different simulation) on windows computer with 20 cores, that didn't crash.
I allso run the simulation using the command "mpiexec -np 8 fds sav88-2b_002.fds", and actually want to use 8 processors when I have 8 mesh
Is the command correct?
In addition, because I am a new user of Linux and since we installed the software on the new computer, we have errors of this type in different runs in different projects.
Is it related to the linux computer? Maybe for the installation of FDS and the software settings?

I would appreciate your help in understanding the error.
thanks
Mor
[fds file.txt](https://github.com/user-attachments/files/16492903/fds.file.txt)

**Comment by jfpe37** [comment]
Hi
I would like to know why the discussion was closed ?
I would appreciate any help regarding the error I received
Thanks


מור קיטל, ראש מחלקת סימולציות
     טלפון: 03-7369772 שלוחה: 124
     נייד: 054-2576192
[b4aefd71-5307-4e73-aa77-407b29a43f72]
י. פרידמן הנדסת בטיחות אש
עמק זבולון 13 א', גני-תקוה
ת. ד. 4232, מיקוד 5592912

**Reply by mcgratta** [reply]
This discussion became an issue post

**Comment by mcgratta** [comment]
This Discussion post became an [issue](https://github.com/firemodels/fds/issues/13283) so that someone could be assigned to work on it.  Otherwise we run the risk of having multiple people work on it needlessly.
